README.md

```diff
@@ -11,7 +11,23 @@ fits into the file check workflow.
 [adr]: https://github.com/nationalarchives/tdr-dev-documentation/blob/master/architecture-decision-records/0013-file-check-queues-and-lambdas.md
 
 ### Adding new environment variables to the tests
-The environment variables in the deployed lambda are encrypted using KMS and then base64 encoded. These are then decoded in the lambda. Because of this, any variables in `src/test/resources/application.conf` which come from environment variables in `src/main/resources/application.conf` need to be stored base64 encoded. There are comments next to each variable to say what the base64 string decodes to. If you want to add a new variable you can run `echo -n "value of variable" | base64 -w 0` and paste the output into the test application.conf
+The environment variables in the deployed lambda are encrypted using KMS and then base64 encoded. These are then decoded
```

Comment:
Did you mean to put a line break in here?

Reply:
Yeah, I find line breaks in Markdown files are easier to review on GitHub and some IDEs, so I tend to break up lines like this. It avoids long horizontal scrolling, and it's easier to leave a review comment on a specific part of a paragraph. The line breaks aren't visible in the rendered document.

I know we've got a couple of different styles in our Markdown files, but it hasn't seemed like a big enough deal to actually have a discussion about the preferred style yet. There are pros and cons to both: https://about.gitlab.com/blog/2016/10/11/wrapping-tet/

If you have a strong preference, we can ask the rest of the team and pick one. 🙂

Reply:
No I don't mind. I forgot that they don't show up in the rendered markdown so I'm happy.